### DISCUSSION TITLE
Welcome to lovelace-animated-background Discussions!

### DISCUSSION THREAD

**Discussion by dreimer1986** [dreimer1986/lovelace-animated-background#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by leonardo20022002** [comment]
Hi, I'm writing to you because since the last Home Assistant update, unfortunately the animated background with your integration no longer works and I haven't found any videos online that can solve everything and I wanted to ask you if you could tell me how to fix the problem because I don't know how to do it .I'll explain briefly what I did basically I redid the installation without uninstalling the repository I used the hacs method and it doesn't seem to work can you help me out? Thank you

**Reply by leonardo20022002** [reply]
![image](https://user-images.githubusercontent.com/103932215/235545058-101f3bff-09e6-45b2-9db2-782a0aadcc5b.png)

**Comment by dreimer1986** [comment]
So you removed the Original Villhelm Repository one that is by default in HACS and added https://github.com/dreimer1986/lovelace-animated-background as custom repositry AND installed this one?

If so, push F12 and do a right click on the refresh button. "Clear cache and completely update" or something like this is the way to go.

**Comment by leonardo20022002** [comment]
then no I didn't uninstall anything from hacs but I just put the parameters
back if you want I'll send you photos in such a way so if you want you can
understand better

Il giorno mar 2 mag 2023 alle ore 10:20 Daniel Reimer <
***@***.***> ha scritto:

**Comment by leonardo20022002** [comment]
[image: image.png] I reinstalled the repository version

Il giorno mar 2 mag 2023 alle ore 10:20 Daniel Reimer <
***@***.***> ha scritto:

**Reply by dreimer1986** [reply]
Your image does not show up

**Reply by leonardo20022002** [reply]
![image](https://user-images.githubusercontent.com/103932215/235617711-f8f7b305-da96-4861-ad86-9c29d612758a.png)

**Comment by leonardo20022002** [comment]
if he wants, I'll give him the credentials and he'll fix it for me, I don't
know

Il giorno mar 2 mag 2023 alle ore 10:20 Daniel Reimer <
***@***.***> ha scritto:

**Reply by dreimer1986** [reply]
First uninstall the current Addin from HACS. Then go to HACS -> Frontend and then
![image](https://user-images.githubusercontent.com/29841368/235617997-2423bdaa-b39e-44b5-bbcf-fffc0ead5ab5.png)
in the top right corner. There click on something like custom respositories:
![image](https://user-images.githubusercontent.com/29841368/235618266-e895635a-539d-445b-a77e-8d5c36fc5c66.png)
Add https://github.com/dreimer1986/lovelace-animated-background as you can see there. Likely the version from my repository will then be shown as new green highlighted entry. Install that one. Then try again what will happen then.

**Reply by leonardo20022002** [reply]
OK I try

**Reply by leonardo20022002** [reply]
ok then it is already installed
![image](https://user-images.githubusercontent.com/103932215/235620234-16e9d6a9-098d-4162-ad38-ec25ed8bdde7.png)

**Reply by dreimer1986** [reply]
I get this when I add it:
![image](https://user-images.githubusercontent.com/29841368/235620615-8c1f03b6-31f6-40c8-a38e-b0757494d8d2.png)
If you already have it, then you should find two of these in the search.

**Reply by leonardo20022002** [reply]
I can't find a single one

**Reply by dreimer1986** [reply]
down below is the search button in blue.
![image](https://user-images.githubusercontent.com/29841368/235621318-86a329c7-26e5-4b10-a6d0-e66a0d159f3f.png)
Search for animated and there are two repos. I guess it should be the last one. At least here it is.

**Comment by leonardo20022002** [comment]
How do I insert its repository in HACS? maybe I'm wrong the repository that I install most probably because I install this: https://github.com/Villhellm/lovelace-animated-background but because if I write the name of the repository I only find this

**Reply by dreimer1986** [reply]
Yes, this is the old one. Mine is not in HACS by default as I don't really plan to continue maintaining it. I fought my way though the to me unknown language Java Script and made my own replacement which bypasses most possible future breakages by adding the video on the lowest layer of the site. This makes it rock solid, but needs some help in form of theme editing. This solution is not so easy to be added and configured, but I mainly made it for myself anyways. ^^ https://github.com/dreimer1986/yourname_card_mod/blob/master/README-script.md

**Reply by leonardo20022002** [reply]
ok so what should I do, should I remove the existing one and put this one you gave me?

**Reply by dreimer1986** [reply]
Excatly. No Villhellm Repository version, but the dreimer1986 one.

**Reply by leonardo20022002** [reply]
![image](https://user-images.githubusercontent.com/103932215/235621351-41477e41-cbc4-40d7-bdaa-14fe2887c3cf.png)
is this right?

**Reply by leonardo20022002** [reply]
ok everything works thank you very much in any case if i have problems can i always write you here?

**Reply by dreimer1986** [reply]
if it says dreimer1986 above left, then yes.

**Reply by dreimer1986** [reply]
Sure, I don't plan to add new stuff, but if there are problems, ping me. ^^

**Reply by leonardo20022002** [reply]
ok tank you bye
